ONSdigital/sdx-transform-cs
# Handle extra comment in QBS

## What? and Why?
QBS is having a comment field added to it (qcode 146).  This PR handles this new qcode and makes sure it ends up in the PCK.

## How to test
 - Check out this branch and sdx-console (branch has the same name as this one)
 - Put through a 139.0001 submission.  You'll notice this has a comment field in the example.
 - verify that the comment appears in the image and pck file.

## Checklist
